Hackney.Shared.Tenure/Sns/EventData.cs

```diff
@@ -0,0 +1,8 @@
+namespace Hackney.Shared.Tenure.Sns
+{
+    public class EventData
+    {
+        public object OldData { get; set; }
+        public object NewData { get; set; }
+    }
+}
```

Comment:
Not needed here. The Sns event object is common but should not go in this package.